ci-operator/step-registry/cucushift/installer/check/vsphere/cpms/cucushift-installer-check-vsphere-cpms-commands.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+if test -f "${SHARED_DIR}/proxy-conf.sh"
```

Comment:
I think for vpshere, we do not need this if block.

Reply:
ok. will delete